### PR TITLE
JBIDE-20431 Make sure all installed features...

### PR DESCRIPTION
**Pull request by nickboldt**
JBIDE-20431 Make sure all installed features are shown in Help->About

Signed-off-by: nickboldt <nboldt@redhat.com>

### CODE REVIEW

**Comment by nickboldt** [comment]
testPR

File: plugins/org.jboss.tools.jst.angularjs/about.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+blurb=JBoss Tools Java Standard Tools AngularJS\n\nVersion\: {featureVersion}\n\n(c) Copyright (c) Red Hat, Inc., contributors and others 2013.  All rights reserved.\nVisit http\://jboss.org/tools
```
Comment (jeffmaury) [inline plugins/org.jboss.tools.jst.angularjs/about.properties]: Don't understand what Java is doing here. It would rather have:
JBoss Tools JavaScript Tools for AngularJS
to be in line with org.jboss.tools.jst.js

File: plugins/org.jboss.tools.jst.jsdt/about.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+blurb=JBoss Tools Java Standard Tools Tern.java Adapter\n\nVersion\: {featureVersion}\n\n(c) Copyright (c) Red Hat, Inc., contributors and others 2013.  All rights reserved.\nVisit http\://jboss.org/tools
```
Comment (jeffmaury) [inline plugins/org.jboss.tools.jst.jsdt/about.properties]: Idem. would rephrase as:
JBossTools JavaScript Tools Tern.java Adapter
